wgpu: Prefix WGPU labels with "vello."

waywardmonkeys opened the pull request:
This makes them easier to identify as being from Vello when embedding into an application.

waywardmonkeys commented:
For example, in Apple's tools:

![image](https://github.com/user-attachments/assets/27a554db-8070-4c62-9c03-b4c75d735219)

But when you have your own application and a bunch of pipelines, then it is harder to identify who something belongs to without it being clear in the name.

DJMcNab reviewed (approved):
Would it be worth automatically stripping these prefices in `with_winit` for display?

Otherwise, seems reasonable. Should we also apply this for buffers as well?

waywardmonkeys commented:
Maybe and maybe. I confess to feeling lazy about the first. The second, less so ... but I hadn't run into it yet in the tools. :) Will think about it for a couple of hours!

DJMcNab commented:
Laziness about the first is very reasonable, fwiw.

waywardmonkeys commented:
I changed this to use `vello.` as a prefix instead of `vello_`.

I updated it to handle various buffers as well.

For debug stuff, I prefixed with `vello.debug.`

In some cases, I changed a space character to an underscore.

DJMcNab commented on vello/src/render.rs:
```diff
@@ -178,9 +178,9 @@ impl Render {
+            recording.upload_uniform("vello.config", bytemuck::bytes_of(&cpu_config.gpu)),
         );
         let info_bin_data_buf = ResourceProxy::new_buf(
             buffer_sizes.bin_data.size_in_bytes() as u64,
```
There are lots of other calls to `new_buf`; why have you changed only the ones you have?

waywardmonkeys replied:
because I wasn't looking at new_buf but `upload` / `upload_uniforms` ... I'll hit the others now, I guess. Thanks!

waywardmonkeys replied:
Fixed and force pushed.

waywardmonkeys commented:
One more thing it seems ... looking at it now under the Metal tools.

DJMcNab reviewed (approved):
This will accrue merge conflicts, and seems fine to land
